packages/mobile/src/components/TokenTotalLineItem.tsx

```diff
@@ -44,11 +49,17 @@ export default function TokenTotalLineItem({
           <Text style={styles.exchangeRate} testID="TotalLineItem/ExchangeRate">
             {tokenInfo?.symbol}
             {' @ '}
-            <TokenDisplay
-              amount={new BigNumber(1)}
-              tokenAddress={tokenAddress}
-              showLocalAmount={true}
-            />
+            {localAmount?.exchangeRate ? (
+              `${
+                LocalCurrencySymbol[localAmount.currencyCode as LocalCurrencyCode]
+              }${formatValueToDisplay(new BigNumber(localAmount.exchangeRate))}`
```

Comment:
Why not using `CurrencyDisplay` here?
Looks like there's an opportunity to have a component that can display both Token and CurrencyCode values.

Reply:
Honestly because it's too complex, it has too many things and it's hard to see which transformations will be made to the amount sent to it, in this case there's one number and we want to print the number as-is, with no transformations. This is a "localAmount" but both `CurrencyDisplay` and `TokenDisplay` have rules to multiply by exchange rates so since it was such a simple use case instead fo adding special rules to them I figured it was simpler to just print the item directly.

Reply:
Ok I see, I just don't like too much to multiply the ways we display monetary amounts. Would prefer to have all places wrapped in a function call or component so it's easier to change in the future.
I guess here's since we're using `formatValueToDisplay` it's fine.

Reply:
I agree there should be just one, I just really did feel that this was the solution that reduced the added complexity the most, but I would prefer to go through `TokenDisplay` in most situations